pub_sub/csharp/order-processor/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();

```

Comment:
Do we need swagger for this sample? it seems like an extra we could take out.

Reply:
you're right lets reduce concepts.